src/DataMigration/custom/Helpers/Assessment.ps1

```diff
@@ -0,0 +1,59 @@
+#-----------------------------------------------------------------------
```

Comment:
```suggestion
# ----------------------------------------------------------------------------------
#
# Copyright Microsoft Corporation
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
# http://www.apache.org/licenses/LICENSE-2.0
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
# ----------------------------------------------------------------------------------
```

Reply:
If you want to use a copyright header, please use the header of Apache License.
The same with the other files.